client/src/modules/user/UserProfile/EmailNotifications.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { PureComponent } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import AlertBox from 'common/components/AlertBox';
+import { saveEmailNotificationSettings } from '../model/actions';
+import { NotificationFrequency, MessageType } from 'common/constants/enums';
+import SelectDays from './SelectDays';
+import './EmailNotification.scss';
+
+// TODO: Fetch settings when profile view is displayed
```

Comment:
Hmm I think in this task, all features connected to notification settings should be done... (front, backend, getting and saving)  The method saving notification settings in DB is created, why fetching setting is missing? I think when we get user details we should send only one prop more?